system/api-server/src/file-upload/file-upload.service.ts

```diff
@@ -88,7 +88,7 @@ export class ImageUploadService {
     try {
       const v = await this.fileUploadRepository
       await v.find().then((data) => {
-        return res.status(201).json({ 
+        return res.status(200).json({ 
```

Comment:
Wouldn't this be a 200 request since it's retrieving resources rather than signaling it's been created successfully?

Reply:
Yes, it should be a 200 since the request was successful